relayer/chains/wasm/client.go

```diff
@@ -259,7 +258,7 @@ func (c *Client) Reconnect() error {
 }
 
 // Signs the message with private key
-func (c *Client) SignMessage(uid string, message []byte) ([]byte, error) {
-	signedMsg, _, err := c.ctx.Keyring.Sign(uid, message, signing.SignMode_SIGN_MODE_DIRECT)
-	return signedMsg, err
+// TODO: not sure if this is the right way to sign the message
+func (c *Client) SignMessage(uid string, message []byte) []byte {
```

Comment:
Is this still required after the new GetSignMessage method ?

Reply:
yeah, this is redundant now.